src/app/landing/LandingPage.jsx

```diff
@@ -112,17 +127,26 @@ class LandingPage extends Component {
           <div className="TestimonialsInner">
             <h3 className="TestimonialsTitle">GoGettas are saying</h3>
             <Testimonial
-              img="img/landing_page/testimonial_jeffrey.png"
+              img={{
+                small: 'img/landing_page/testimonial_jeffrey_mobile.png',
```

Comment:
If GitHub is to be trusted, these testimonials on mobile aren't actually smaller than the normal versions (~25KB in both cases).
In general, worth reviewing all the file sizes and seeing the mobile is actually an improvement in all cases.